configs/envs/.env.eth

```diff
@@ -36,7 +36,7 @@ NEXT_PUBLIC_GRAPHIQL_TRANSACTION=0xf7d4972356e6ae44ae948d0cf19ef2beaf0e574c18099
 NEXT_PUBLIC_API_SPEC_URL=https://raw.githubusercontent.com/blockscout/blockscout-api-v2-swagger/main/swagger.yaml
 NEXT_PUBLIC_HAS_BEACON_CHAIN=true
 NEXT_PUBLIC_IS_ACCOUNT_SUPPORTED=true
-NEXT_PUBLIC_AUTH_URL=http://localhost:3000
+# NEXT_PUBLIC_AUTH_URL=http://localhost:3000
```

Comment:
?

Reply:
atm, yup thinks that localhost url is not valid url, so I have to comment this out.. auth doesn't work loacally anyways